packages/ui/ProgressBar/ProgressBar.js

```diff
@@ -57,29 +57,29 @@ const ProgressBar = kind({
 		css: PropTypes.object,
 
 		/**
-		 * The proportion of the filled portion of the progress bar. Valid values are
-		 * between `0` and `1`.
+		 * Sets the orientation of the slider, whether the progress-bar depicts its progress value
+		 * in a left and right orientation or up and down onientation.
+		 * Must be either `'horizontal'` or `'vertical'`.
 		 *
-		 * @type {Number}
-		 * @default 0
+		 * @type {String}
 		 * @public
 		 */
-		progress: PropTypes.number,
+		orientation: PropTypes.oneOf(['horizontal', 'vertical']),
 
 		/**
-		 * If `true` the progress bar will be oriented vertically.
+		 * The proportion of the filled portion of the progress bar. Valid values are
+		 * between `0` and `1`.
 		 *
-		 * @type {Boolean}
-		 * @default false
+		 * @type {Number}
+		 * @default 0
 		 * @public
 		 */
-		vertical: PropTypes.bool
+		progress: PropTypes.number
 	},
 
 	defaultProps: {
 		backgroundProgress: 0,
-		progress: 0,
-		vertical: false
+		progress: 0
```

Comment:
Should we have a default here, as we do in Moonstone?

Reply:
Yes, probably. At least until we open up `orientation` in UI.